data/jobs.txt

```diff
@@ -1483,6 +1483,238 @@ mission "Core Pirate Attack"
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
 
+mission "Renegade Drones Intro"
+	name "Neutralize broken Drone"
+	description "Destroy or disable the malfunctioning Navy drone."
+	deadline 1
+	minor
+	to offer
+		random < 20
+		"combat rating" > 200
+	source "Pilot"
+	on offer
+		conversation
+			`Walking past a group of new Navy recruits, you hear one say, "One more of our drones seems to have malfunctioned in orbit, attacking any ship on its scanners. We need to take care of it before our commanding officers find out about this. Whose turn is it this time?"`
+			choice
+				`	(Raise your hand and offer to shoot it down.)`
+					goto yes
+				`	(Ignore them and walk on.)`
+					defer
+			label yes
+			`You raise your hand and one of the soldiers notices you.`
+			`	"You? Why not. Try not to hurt yourself."`
+			`	You got the job.`
+				launch
+	npc evade
+		government "Renegade"
+		personality waiting heroic target vindictive marked
```

Comment:
- You were saying the Renegades are supposed to hate everybody: Shoot on sight.
- `marked` personality prevents the ships from attacking or being attacked by anybody but the player.
Just something to consider.